code/game/mecha/mecha_topic.dm

```diff
@@ -306,6 +307,24 @@
 		maint_access = !maint_access
 		send_byjax(src.occupant,"exosuit.browser","t_maint_access","[maint_access?"Forbid":"Permit"] maintenance protocols")
 
+	if (href_list["toggle_port_connection"])
+		if(internal_tank.connected_port)
+			if(internal_tank.disconnect())
+				occupant_message("Disconnected from the air system port.")
+				log_message("Disconnected from gas port.")
+			else
+				occupant_message("<span class='warning'>Unable to disconnected from the air system port!</span>")
+				return
+		else
+			var/obj/machinery/atmospherics/components/unary/portables_connector/possible_port = locate(/obj/machinery/atmospherics/components/unary/portables_connector) in loc
```

Comment:
`= locate() in loc`

Reply:
Same line occurs in portable_atmospherics.dm. Should i fix it there?